headings.html

```diff
@@ -0,0 +1,46 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
```

Comment:
Thanks, I think the color was originally included but was not needed.  Will remove.